plugins/arrow-flight-rpc/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -0,0 +1,45 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+grant codeBase "${codebase.netty-common}" {
```

Comment:
I believe `grant codeBase` does not work for plugins :(

Reply:
it runs into permission issue if i remove explicit `grant codeBase`, so i believe it works.